xml/SLES-JeOS-12SP4-Quickstart.xml

```diff
@@ -294,23 +293,10 @@
       <para>24Gb Linux filesystem </para>
      </listitem>
     </itemizedlist>
-    <para>For additional disk space, add a secondary disk to the VM.</para>
-   </sect3>
-
-
-   <sect3 xml:id="sec-full-disk-layout-description">
-    <title>Full Disk Layout Description</title>
-
-    <para>The kiwi configuration files used to create the official SUSE Linux
-     Enterprise Server JeOS images are available from the openSUSE Build Service at
-     <link xlink:href="https://build.opensuse.org">https://build.opensuse.org</link>.
-     For example you can check the disk layouts for SUSE Linux Enterprise Server
-     15 JeOS:</para>
-     <para>
-     <link xlink:href="https://build.opensuse.org/package/view_file/SUSE:SLE-15:GA/kiwi-templates-SLES15-JeOS/JeOS.kiwi?expand=1"/></para>
-     <para><emphasis role="strong"> &lt;(REPLACE SLE15 WITH SLE12SP4 LINKS)&gt;</emphasis></para>
-   </sect3>
+    <para>For additional disk space, you should be able to enlarge the existing
```

Comment:
Can be done, but the extra space would have to be partitioned and formatted manually.